### ISSUE TITLE
Flow Feather y Flow Fight

### ISSUE THREAD

**Issue by ConiiiH** [franciscogazitua/IIC2113-2024#105]
Holaa! Estoy teniendo problemas con los test cases E42 15 y 17,
En los test cases no se activa la segunda parte de los efectos porque no se cumple la condicion spd unidad >= spd rival - 10. Sin embargo segun mis calculos si se cumple, en el 15 eliwood tiene spd 45 y alear 54. en el 17 micaiah tiene 39 e ike 41. Entonces no entiendo porque no se debe agregar el daño extra y la reduccion de daños en estos casos.

<img width="480" alt="Captura de pantalla 2024-06-16 a la(s) 7 47 08 p  m" src="https://github.com/franciscogazitua/IIC2113-2024/assets/95378081/da7c7154-ea02-45ae-87f5-399774b6df0f">
<img width="480" alt="Captura de pantalla 2024-06-16 a la(s) 7 48 07 p  m" src="https://github.com/franciscogazitua/IIC2113-2024/assets/95378081/e907ec69-5c56-4df3-8700-2b2c9099c693">

**Comment by RodrigoToroIcarte** [comment]
Creo que es porque la diferencia en Res (o en Def) es cero en ambos casos. Entonces no pasa nada en la práctica (aunque se active la segunda parte del efecto).

**Comment by ConiiiH** [comment]
en el primer caso se tiene:
Eliwood res: 26
Alear res 33
la diferencia es negativa, por eso es 0?
creo que estaba sacando el valor absoluto

**Comment by RodrigoToroIcarte** [comment]
Sí. La habilidad dice que el Min es cero y el Max es 7. Por eso si la resta es negativa te queda que el daño extra/reducción es igual a cero. Y por eso no se muestra el mensaje (a pesar de que, técnicamente, sí ocurrió el segundo efecto).

**Comment by ConiiiH** [comment]
Con eso me funciono!!
Muchas gracias profee :)
